### ISSUE TITLE
[dev.icinga.com #7972] Provide text descriptions to identify required fields that were not completed

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#1242]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/7972

**Created by mhein on 2014-12-03 11:50:29 +00:00**

Assignee: _jmeyer_
Status: _Resolved_  (closed on _2015-02-12 08:13:56 +00:00_)
Target Version: _2.0.0-beta3_
Last Update: _2015-02-12 08:13:56 +00:00 (in Redmine)_

---

http://www.w3.org/TR/WCAG20-TECHS/G83.html wrote:
> The objective of this technique is to notify the user when a field that must be completed has not been completed. When users fail to provide input for any mandatory form fields, information is provided in text to enable the users to identify which fields were omitted. One approach is to use client-side validation and provide an alert dialog box identifying the mandatory fields which were omitted. Another approach, using server-side validation, is to re-display the form (including any previously entered data), with either a text description at the location of the omitted mandatory field, or a text description that identifies the omitted mandatory fields. \[...\]

Currently we only show "Value is required and can't be empty". We must show the labels of the fields that were not completed.

---

**Parent Task:** [#8415](https://dev.icinga.com/issues/8415)
